lib/transaction.ex

```diff
@@ -31,6 +31,73 @@ defmodule Elixium.Transaction do
     %{outputs: outputs}
   end
 
+  @doc """
+  Creates a correct tx id
+  """
+  @spec create_tx_id(List) :: String.t()
+  def create_tx_id(tx) do
+    calculate_hash(tx)
+    |> Utilities.sha_base16()
```

Comment:
No need to sha_base16 at all, the result of calculate_hash/1 is already a base16 string. I think there may be some confusion here. The calculate_hash function's sole purpose is to calculate the tx id, so this function shouldn't exist. It might make sense to rename the calculate_hash function to calculate_tx_id

Reply:
Removed & Updated Both Tests & Functions